docs/mivot/viewer.rst

```diff
@@ -173,25 +257,27 @@ identifiers, which have the following structure: ``model:a.b``.
 - Original ``@dmtype`` are kept as attributes of generated Python objects.
 - The structure of the ``MivotInstance`` objects can be inferred from the mapped model in 2 different ways:
 
-  - 1.  From the MIVOT instance property ``MivotInstance.dict`` a shown above.
+  - 1.  From the MIVOT instance property ``MivotInstance.to_dict()`` a shown above.
         This is a pure Python dictionary but its access can be slow because it is generated
         on the fly each time the property is invoked.
   - 2.  From the internal  class dictionary ``MivotInstance.__dict__``
         (see the Python `data model <https://docs.python.org/3/reference/datamodel.html>`_).
 
- .. code-block:: python
-    :caption: Exploring the MivotInstance structure with the internal dictionaries
+.. code-block:: python
+
+   mivot_instance = mivot_viewer.dm_instance
+
+   print(mivot_instance.__dict__.keys())
+   dict_keys(['dmtype', 'longitude', 'latitude', 'pmLongitude', 'pmLatitude', 'epoch', 'Coordinate_coordSys'])
```

Comment:
Probably worth commenting these out to make it easier to cut, paste and use the example:

```suggestion
   # dict_keys(['dmtype', 'longitude', 'latitude', 'pmLongitude', 'pmLatitude', 'epoch', 'Coordinate_coordSys'])
```

Also, using the query `I/239/hip_main` query from above, I don't see the same keys expected here.  If that was the example intended here, then maybe these could be updated, though keeping this sort of example up to date could be challenging.